docs/BR.md

```diff
@@ -1935,44 +1944,60 @@ If the Subordinate CA is not allowed to issue certificates with dNSNames, then t
+Effective 2020-09-30, a Certificate issued to a Subscriber MUST contain, within the Certificate's certificatePolicies extension, one or more policy identifier(s) that are specified beneath the CA/Browser Forum's reserved policy OID arc of `{joint-iso-itu-t(2) international-organizations(23) ca-browser-forum(140) certificate-policies(1)} (2.23.140.1)`.
 
 The certificate MAY also contain additional policy identifier(s) defined by the Issuing CA. The issuing CA SHALL document in its Certificate Policy or Certification Practice Statement that the Certificates it issues containing the specified policy identifier(s) are managed in accordance with these requirements.
 
 For certificates issued prior to 2020-09-30, a Certificate issued to a Subscriber MUST contain a certificatePolicies extension. The extension MUST contain one or more policy identifiers that indicate adherence to and compliance with these Requirements. CAs MUST either use a CA/Browser Forum identifier reserved for this purpose or MUST use a policy identifier documented by the CA in its Certificate Policy and/or Certification Practice Statement to indicate the Certificate's compliance with these Requirements.
 
+Prior to including a Reserved Certificate Policy Identifier, the CA MUST ensure the following requirements are met:
+
+* __Certificate Policy Identifier:__ `2.23.140.1.2.1`  
+  If the Certificate complies with these requirements and lacks Subject identity information that has been verified in accordance with Section 3.2.2.1 or Section 3.2.3.  
+  Such Certificates MUST NOT include `organizationName`, `givenName`, `surname`, `streetAddress`, `localityName`, `stateOrProvinceName`, or `postalCode` in the Subject field.  
+* __Certificate Policy Identifier:__ `2.23.140.1.2.2`  
+  If the Certificate complies with these Requirements and includes Subject Identity Information that is verified in accordance with Section 3.2.2.1.  
+  Such Certificates MUST also include `organizationName`, `localityName` (to the extent such field is required under Section 7.1.4.2.2), `stateOrProvinceName` (to the extent such field is required under Section 7.1.4.2.2), and `countryName` in the Subject field.  
+* __Certificate Policy Identifier:__ `2.23.140.1.2.3`  
+  If the Certificate complies with these Requirements and includes Subject Identity Information that is verified in accordance with Section 3.2.3.  
+  Such Certificates MUST also include either `organizationName` or both `givenName` and `surname`, `localityName` (to the extent such field is required under Section 7.1.4.2.2), `stateOrProvinceName` (to the extent required under Section 7.1.4.2.2), and `countryName` in the Subject field.  
```

Comment:
```suggestion
  Such Certificates MUST also include the following in the Subject field:
  (i) `organizationName`, or
  (ii) both `givenName` and `surname`, or
  (iii) both (i) and (ii)
  
  Additionally, Certificates MUST also include `localityName` (to the extent such field is required under Section 7.1.4.2.2), `stateOrProvinceName` (to the extent required under Section 7.1.4.2.2), and `countryName` in the Subject field.  
```

Reply:
Hrm, this seems substantially different (and formats terribly) that I'm going to reject this suggestion. I think, in the broader context, the problem you highlight regarding concern extends well beyond this particular sentence, and so I'm broadly OK with preserving the status quo here and seeing this tackled separately (perhaps as part of certificate profiles), given the inter-relationships.